debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -644,6 +644,11 @@ public static AutoCreateMode parse(String value, String defaultValue) {
                             "FULL - Records the old values of all columns in the row." +
                             "NOTHING - Records no information about the old row. This is the default for system tables.");
 
+    public static final Field REPLICA_IDENTITY = Field.create("replica.identity")
```

Comment:
Why is this new config required ?

Reply:
This is not required now, removing it from the PR.